packages/cashier-v2/src/lib/PaymentAgent/components/PaymentAgentCardDescription/PaymentAgentCardDescription.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import type { IconTypes } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui';
+import type { THooks } from '../../../../hooks/types';
+import { capitalizeFirstLetter } from '../../../../utils';
+import { paymentMethodIcons } from '../../constants';
+import { getNormalizedIconPaymentMethod } from '../../utils';
+import styles from './PaymentAgentCardDescription.module.scss';
+
+type TProps = {
+    paymentAgent: THooks.PaymentAgentList[number];
+};
+
+const PaymentAgentDepositCardDescription: React.FC<TProps> = ({ paymentAgent }) => {
+    const {
+        further_information: furtherInformation,
+        name,
+        supported_payment_methods: supportedPaymentMethods,
+        urls,
+    } = paymentAgent;
+
+    const theme: 'dark' | 'light' = 'light';
+
+    const icons = supportedPaymentMethods.reduce((acc, { payment_method: paymentMethod }) => {
+        if (!paymentMethod) return acc;
+        const normalizedIconPaymentMethod = getNormalizedIconPaymentMethod(paymentMethod);
+        if (!normalizedIconPaymentMethod) return acc;
+        if (!acc.has(paymentMethodIcons[normalizedIconPaymentMethod][theme])) {
+            acc.add(paymentMethodIcons[normalizedIconPaymentMethod][theme]);
+        }
+        return acc;
+    }, new Set<{ icon: IconTypes; key: string }>());
+
+    return (
+        <div className={styles.container}>
+            <div className={styles['info-container']}>
+                <Text as='p' size='sm' weight='bold'>
+                    {name}
+                </Text>
+                {furtherInformation && (
+                    <Text as='p' size='sm'>
+                        {capitalizeFirstLetter(furtherInformation).replace(/( ?Skril?l,? ?)|( ?Net?tel?ler,? ?)/gi, '')}
```

Comment:
whoa that looks suspisious, replacing parts of string with regexpes? 
not sure about context, but I guess ideally we shouldn't do that (if its from backend - we should get the proper text immediately)

Reply:
basically - we shouldn't process information like that on frontend especially given the fact that its probably language-specific

Reply:
hey @wojciech-deriv 
yes I completely agree with you and there was a discussion regarding this around 1 year ago.
there was a request to remove all spellings of `Skrill` and `Netteller` from payment agent description per Paysafe mandate.
because of compliance team was not be able do it manually in each payment agent description we did it on FE side.
actually it's a good point to check with BE regarding adding such regex check on their side.
i will raise this
thanks for highlighting this @wojciech-deriv